audio/FAudio/Portfile

```diff
@@ -18,24 +18,27 @@ long_description        an XAudio reimplementation that focuses solely on develo
 
 depends_lib-append      port:libsdl2
 
-checksums               rmd160  eca88c3b235e6815c7c17bf6392c2ffdc30d9a25 \
-                        sha256  5846d61f088cd7b4450e6424a21bc4cff8f6074ab0cee702586b2a3b784bfdf6 \
-                        size    1131873
+checksums               rmd160  e24c9e179d5a806ad63d324a63a4200f6a6b321e \
+                        sha256  8aef339e0ddef9ca963a2903407e6ed254dbf8427fdb2977df6f47e99047f700 \
+                        size    1131793
 
 # remove set deployment target and hard-coded RPATH setting
 patchfiles              patch-faudio-remove-deployment-target.diff
 
-configure.args          -DFFMPEG=OFF \
+configure.args          -DGSTREAMER=OFF \
                         -DBUILD_UTILS=OFF \
                         -DBUILD_TESTS=ON \
                         -DXNASONG=ON
 
-variant ffmpeg description "Use ffmpeg for additional xWMA support - note license changes" {
+variant wma description "Use gstreamer1-gst-libav for additional xWMA support - note license changes" {
     license                GPL-2+
```

Comment:
Why does this license change happen? Are we sure that it still happens now that we've switched to gstreamer or was it specific to having used ffmpeg? If the license change still happens, are we sure that the entire project then becomes GPL-2+ licensed (that's what is being specified here but seems unlikely to me) or is it just that we must obey GPL-2+ in addition to the previously specified license (seems more likely to me)?

Reply:
I’m not sure on this one since gst-libav uses ffmpeg for wma decoding see https://github.com/GStreamer/gst-libav#plugin-dependencies-and-licenses

As we had this for directly using ffmpeg I’m assuming it should still be the same